tests/config_id/README.md

```diff
@@ -0,0 +1,12 @@
+## Open Enclave Init-time Configuration Interface.
+This document describes how developers can set CONFIGID and CONFIGSVN fields from the host side software at enclave initialization time and retrieve same information on enclave software side.
+[Overview of SGX CONFIGID and CONFIGSVN](https://github.com/openenclave/openenclave/blob/cd72fd7069488ba6f453c8f5f47bd9fd9a6e6c0d/docs/DesignDocs/InitTimeConfigurationInterface.md#sgx-configid-and-configsvn-overview).
+#### Setting CONFIGID and CONFIGSVN at enclave initialization time
+
+CONFIGID and CONFIGSVN information is passed in to the enclave creation settings argument with setting type `OE_SGX_ENCLAVE_CONFIG_DATA` and configuration data with data type `oe_sgx_enclave_setting_config_data`. Line number 39-47 [here](https://github.com/openenclave/openenclave/blob/master/tests/config_id/host/host.c) demonstrates how CONFIGID and CONFIGSVN is filled in `oe_sgx_enclave_setting_config_data` type and setting is passed to enclave creation time [here]('https://github.com/openenclave/openenclave/blob/master/tests/config_id/host/host.c#L61').
```

Comment:
avoid use of "here" in link text per https://www.w3.org/TR/WCAG10-HTML-TECHS/#link-text (in short, for accessibility reasons).  Same in line 8 and 12 below.